www/js/model.js

```diff
@@ -214,7 +242,23 @@ define([
 			},
 			getInitialNodeString: function(){
 				return obj.initialNodeString;
-			}
+			},
+			getInitial: function(/*string*/ id){
+				var node = this.getNode(id);
```

Comment:
there is nothing called initial in the data. It is called value as per the design docs